packages/frontend/app/services/reporting.js

```diff
@@ -116,6 +122,9 @@ export default class ReportingService extends Service {
           'academicYearCrossesCalendarYearBoundaries',
         );
         year = crosses ? `(${record.year} - ${record.year + 1})` : `(${record.year})`;
+      } else if (model === 'program-year') {
```

Comment:
I think this isn't needed. 
 Creating a report for all terms associated with a program year results in:

> This report shows all Terms associated with Program Year "2028 Doctor of Medicine - BRIDGES" **(2028)** in Medicine.

Which puts an extra year at the end, but the program year is already part of the description.

Reply:
I agree that the duplication isn't necessary. The reason it's there is because the translation file uses a format that requires a `(year)` argument and it was adding a blank space otherwise.

https://github.com/ilios/frontend/blob/master/packages/frontend/translations/en-us.yaml#L345

If we don't want the extra `(year)` then wouldn't we need a new translation term solely for this case?

Reply:
For sure don't want the extra year just for that. I think we can live with the space, but I'd be pretty surprised if it wasn't possible to eliminate it in the message if the value wasn't set. Might be worth poking into the ICU message syntax docs and what is in https://ember-intl.github.io/ember-intl/ to see if that is possible.

Reply:
OK, I see where this got "hacked" in another test without creating a new translation or adding duplicate info: https://github.com/ilios/frontend/blob/master/packages/frontend/tests/unit/services/reporting-test.js#L150

You may notice the double space before `in ${school.title}.` that I just did ;P

Not my favorite way to solve this, but this PR is getting long in the tooth so I'm just going to copy that.

Reply:
The title that gets produced naturally reduces the double space to a single space, anyway, because HTML.